core/include/debug.h

```diff
@@ -14,7 +14,7 @@
  * @brief       Debug-header
  *
  * @details     If ENABLE_DEBUG is set, before this header is included,
- *              ::DEBUG will print out to stdout, otherwise do nothing
+ *              ::DEBUG_FUNC will print out to stdout, otherwise do nothing
```

Comment:
I'm afraid I don't get the motivation behind this change... If I understand it correctly, DEBUG_FUNC isn't really the core function of debug.h; or did I miss something?